index.html

```diff
@@ -11,44 +11,38 @@
 <body>
     <div class="maincontainer" >       
 
-        <div class="mainmenu"> 
+        <header  class="mainmenu"> 
 
-            <div class="LogoPlaceholder"> <img src="img/Mylogo.png" alt="my logo">   </div>
+            <div class="LogoPlaceholder"><a href="#index.html" ><img src="img/Mylogo.png" alt="my logo"> </a>  </div>
 
-            <div class="itemnavigation">
-                <div class="navitems "><a href="#portfolio" > Portfolio</a> <br> </div> 
-                
-                <div class="navitems"><a href="#portfolio" > About</a> </div> 
-
-                <div class="navitems" ><a href="#portfolio" > contact</a> </div> 
+            <nav class="itemnavigation">
+                <div class="navitems "><a href="#portfolio" > <img src="img/menu.png" alt="my logo"> </a> <br> </div>                 
+               
                
-            </div> 
-         </div>
+            </nav> 
+        </header>
 
 
        
        <div class="healdline">       
-        <img src="img/Headerbg.png" alt="my logo"> 
-            <br>
-            <br>
-                <div class="headlinebody">
-                    <br>
-                      <br> 
-                      <br>
-                      <br>                                   
+        <img src="img/header_mobile.png" alt="my logo"> 
+            
+            
+                <div class="headlinebody">                                                     
                   
                     <div class="introductionclass">
-                      <p>I’m Samuel. <br>
-                        Glad to see you!</p>
+                      <h1>I’m Samuel.  Glad to see   you!.</h1>                      
+                        
                     </div>
                     <div class="second_primary_text">
                         <p> I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
 
                     </div>
                     
-                        <div calss="letconnect">
+                        
                             <p class="letconnect"> LET’S CONNECT</p>
-                        </div>
+                           
+                       
```

Comment:
- [ ] Dear @samuelkinyanjui, Good job making the changes as requested earlier. But while doing so I notice your design is missing some spacing, Please remember we need to preserve the original design. I suggest looking at the following pictures and making the appropriate changes. 
- [ ] ALso, As suggested earlier, your icons and text content seem a bit smaller than the original design. As front-end engineers, please remember that we are always expected to deliver the exact copy of the template. Please consider making the appropriate changes.
### You design on 375px screen size
![image](https://github.com/samuelkinyanjui/Portfolio-mobile-first/assets/69077061/cffe67e6-9385-465c-b78f-5ffb39a92a24)
### Original design 
![image](https://github.com/samuelkinyanjui/Portfolio-mobile-first/assets/69077061/2963b4b0-2bd0-4cc1-9908-a8777d6eb82d)